pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -52,6 +52,9 @@ const (
 
 func generateManifests(role string, version string) ([]string, error) {
 	var manifests []string
+	if version == "" {
```

Comment:
Is it possible?

Reply:
After double check caller side, we always set it as 'latest' if parameter is empty. This can be removed.